app/controllers/report_controller/reports.rb

```diff
@@ -373,6 +373,7 @@ def check_tabs
         elsif @edit[:new][:model] == "Chargeback"
           unless @edit[:new][:cb_show_typ] &&
                  ((@edit[:new][:cb_show_typ] == "owner" && @edit[:new][:cb_owner_id]) ||
+                  (@edit[:new][:cb_show_typ] == "tenant" && @edit[:new][:cb_tenant_id]) ||
```

Comment:
@aljesusg yes, much better - I am for 1.,and I am suggesting also something like this:
```
 # Check if chargeback field is valid
  def valid_chargeback_fields
    is_valid = false
    if %w(owner tenant tag).include?(@edit[:new][:cb_show_typ]) 
      is_valid = case @edit[:new][:cb_show_typ]
                       when "owner"  then @edit[:new][:cb_owner_id]
                       when "tenant" then @edit[:new][:cb_tenant_id]
                       when "tag"    then @edit[:new][:cb_tag_cat] && @edit[:new][:cb_tag_value]
    end
    is_valid
  end
```
inspired by https://github.com/bbatsov/ruby-style-guide and find on this page  `When assigning the result of a conditional expression to a variable`

maybe this will rubocop accept :)